run-tests.php

```diff
@@ -57,6 +57,8 @@ function main()
 	// Parallel testing
 	global $workers, $workerID;
 
+	define('IS_WINDOWS', substr(PHP_OS, 0, 3) == "WIN");
```

Comment:
I personally think this is cleaner:

`PHP_OS_FAMILY == 'Windows'`

The `substr()` call has always been rather redundant for Windows unless you wanted to differentiate between Windows NT and Windows 9x series.

Either way I'm fine with this change overall

Reply:
PHP_OS_FAMILY is in 7.2, but run-tests is currently compatible with 7.0.

Reply:
> 
> 
> PHP_OS_FAMILY is in 7.2, but run-tests is currently compatible with 7.0.

As this is targeting master shouldn't this be alright? Or is run-tests used version agnostic?

Reply:
Context: https://github.com/php/php-src/pull/3981

We can bump the minimum version requirement, I'm just saying that this is the status quo.